app/routes/forum/components/ForumListEntry.tsx

```diff
@@ -0,0 +1,23 @@
+import { Flex } from '@webkom/lego-bricks';
+import cx from 'classnames';
+import { Link } from 'react-router-dom';
+import styles from './ForumList.css';
+import type { PublicForum } from 'app/store/models/Forum';
+
+const ForumListEntry = ({ subForum }: { subForum: PublicForum }) => {
```

Comment:
Is there any reasoning behind calling it a `subForum` and not just a `forum`? Initially I don't see where the sub comes from - as there are no forums that are related to/children of other forums

Reply:
That's a good point, will rename.